analyzers/nodejs/integration_test.go

```diff
@@ -59,15 +65,24 @@ func TestTestSetup(t *testing.T) {
 	assertProjectFixtureExists(t, "request")
 }
 
-func TestAnalysisOutput(t *testing.T) {
+func TestNodejsAnalsis(t *testing.T) {
 	t.Parallel()
-	for _, proj := range projects {
-		t.Run(proj.Name, func(t *testing.T) {
+	for _, project := range projects {
+		proj := project
+		t.Run("Analysis:"+proj.Name, func(t *testing.T) {
+			t.Parallel()
+			shouldAllowNpmErr := false
+			for _, arg := range proj.Args {
```

Comment:
If this is the only usage of `proj.Args`, you should really store a field with module options instead.

Reply:
This is not the only usage, it's meant to define what is passed in as `Options` and as params to other commands... that being said, if we're turning off reporting checks for now, I think that's fine

Reply:
That sounds like a great idea, but we're not _actually_ doing option parsing here. I think we should go all the way for one or the other: either pass an `Options map[string]interface{}` to make our lives easy or do actual option parsing to test that code path. This half-way point feels like the worst of both worlds.

(If you decide to go the full-blown option parsing route, that will require a minor refactor of `package config` -- it may make your life easier to do that in a follow-up PR.)